Mathlib/Data/Nat/Choose/Multinomial.lean

```diff
@@ -305,6 +306,25 @@ theorem sum_pow [CommSemiring R] (x : α → R) (n : ℕ) :
 end CommSemiring
 end Finset
 
+namespace Nat
+variable {ι : Type*} {s : Finset ι} {f : ι → ℕ}
+
+lemma multinomial_two_mul_le_mul_multinomial :
+    multinomial s (fun i ↦ 2 * f i) ≤ ((∑ i in s, f i) ^ ∑ i in s, f i) * multinomial s f := by
+  rw [multinomial, multinomial, ← mul_sum]
+  refine Nat.div_le_of_le_mul' ?_
```

Comment:
Does gcongr make any of this proof cleaner? Or calc-style?

Reply:
Cleaner, not shorter